site/docs/v1/tech/apis/identity-providers/_link-response-body-base.adoc

```diff
@@ -0,0 +1,45 @@
+[api]
+ifeval::["{base_field_name}" != "identityProviderLink"]
+[field]#identityProviderLinks# [type]#[Array]#::
+The list of Link objects.
+endif::[]
+
+[field]#{base_field_name}.displayName# [type]#[String]#::
+The display name for the user.
+
+[field]#{base_field_name}.identityProviderId# [type]#[UUID]#::
+The unique Id of the identity provider.
+
+[field]#{base_field_name}.identityProviderUserId# [type]#[String]#::
+The Id for the User that is provided by the identity provider.
```

Comment:
Also note, that generally speaking, this is the value we expect to be immutable. This is the value that will allow us to link this user on future logins.